_redirects

```diff
@@ -99,6 +101,18 @@ https://next.docs.getdbt.com/*     https://docs.getdbt.com/:splat 301!
 /docs/global-cli-flags	/reference/global-cli-flags	302
 /docs/graph	/docs/writing-code-in-dbt/jinja-context/graph	302
 /docs/guides/writing-custom-schema-tests    /docs/guides/writing-custom-generic-tests
+/docs/guides/best-practices    /guides/best-practices
+/docs/guides/best-practices#choose-your-materializations-wisely    /guides/best-practices   302
+/docs/guides/best-practices#version-control-your-dbt-project       /guides/legacy/best-practices#version-control-your-dbt-project    302    
+/docs/guides/migration-guide/upgrading-to-v1.0       /guides/migration/versions    302    
+/docs/guides/migration-guide/upgrading-to-0-14-0     /guides/migration/versions    302
+/docs/guides/migration-guide/upgrading-to-0-15-0    /guides/migration/versions    302
+/docs/guides/migration-guide/upgrading-to-0-16-0    /guides/migration/versions  302
+/docs/guides/migration-guide/upgrading-to-0-17-0      /guides/migration/versions    302
+/docs/guides/migration-guide/upgrading-to-0-18-0     /guides/migration/versions    302
+/docs/guides/migration-guide/upgrading-to-0-19-0     /guides/migration/versions    302    
+/docs/guides/migration-guide/upgrading-to-0-20-0     /guides/migration/versions    302    
+/docs/guides/migration-guide/upgrading-to-0-21-0     /guides/migration/versions    302    
```

Comment:
I'm not sure we can. I tried using the below syntax and when trying to access `/docs/guides/migration-guide/upgrading-to-0-21-0` it ended up being redirecting to ` /guides/migration/versions/upgrading-to-0-21-0` which threw a 404.

Would this be the correct syntax?
`/docs/guides/migration-guide/*  /guides/migration/versions/:splat 301`

Reply:
Hm. Are there any migration guides we aren't redirecting? If there aren't, you could just redirect everything with:

`/docs/guides/migration-guide/*    /guides/migration/versions   302`

But this is a minor quibble. Assuming Leona gives the thumbs up on the link question, feel free to merge as-is.

Reply:
It looks like `upgrading-to-0-14-0` through `upgrading-to-19-0` are the only that need to get redirected to `/guides/migration/versions` right now since they currently live in the Older versions directory which will be changed in the future to correct the space in the URL. Updated the PR to reflect this.

Also pinged @runleonarun to get confirmation on the `/docs/guides/best-practices/#version-control-your-dbt-project` redirect confirmation.

Reply:
Fair enough! Sometimes you can't simplify things more!